packages/next-templates/src/app/meldingen/graffiti-melding/form/page.tsx

```diff
@@ -66,182 +62,185 @@ export default function Home() {
-              <FormField invalid={!!errors.mainCategory}>
-                <Paragraph>
+              <section className="voorbeeld-section--spacing">
+                <Heading2>Wat wilt u melden?</Heading2>
+                <Paragraph className="voorbeeld-paragraph-title">Kies een categorie die bij uw melding past.</Paragraph>
+                <FormField invalid={!!errors.mainCategory}>
                   <FormLabel className="voorbeeld-form-label" htmlFor="mainCategory">
                     Hoofdcategorie
                   </FormLabel>
-                </Paragraph>
-                <Select
-                  aria-label="Hoofdcategorie"
-                  defaultValue=""
-                  id="mainCategory"
-                  {...register('mainCategory', {
-                    required: 'Dit is verplicht',
-                  })}
-                  placeholder="Kies een hoofdcategorie"
-                  invalid={!!errors.mainCategory}
-                  className="voorbeeld-svg-arrow"
-                >
-                  <SelectOption disabled value="">
-                    Kies een hoofdcategorie
-                  </SelectOption>
-                  <SelectOption value="1">Option #1</SelectOption>
-                  <SelectOption value="2">Option #2</SelectOption>
-                  <SelectOption value="3">Option #3</SelectOption>
-                  <SelectOption value="4">Option #4</SelectOption>
-                </Select>
-                <FormFieldDescription invalid={!!errors.mainCategory}>
-                  {errors.mainCategory?.message}
-                </FormFieldDescription>
-              </FormField>
-              <FormField invalid={!!errors.subCategory}>
-                <Paragraph>
+                  <Select
+                    aria-label="Hoofdcategorie"
+                    className="voorbeeld-select"
+                    defaultValue=""
+                    id="mainCategory"
+                    {...register('mainCategory', {
+                      required: 'Dit is verplicht',
+                    })}
+                    placeholder="Kies een hoofdcategorie"
+                    invalid={!!errors.mainCategory}
+                  >
+                    <SelectOption disabled value="">
+                      Kies een hoofdcategorie
+                    </SelectOption>
+                    <SelectOption value="1">Option #1</SelectOption>
+                    <SelectOption value="2">Option #2</SelectOption>
+                    <SelectOption value="3">Option #3</SelectOption>
+                    <SelectOption value="4">Option #4</SelectOption>
+                  </Select>
+                  <FormFieldDescription invalid={!!errors.mainCategory}>
+                    {errors.mainCategory?.message}
+                  </FormFieldDescription>
+                </FormField>
+                <FormField className="voorbeeld-formField" invalid={!!errors.subCategory}>
```

Comment:
formField maar als je naar de classname kijkt van de formLabel is het form-label. Dus dan is de vraag wil je de classname met camelcasing of met een streep?

andere classnames heb ik ook zo gezien